src/Core/Content/Product/DataAbstractionLayer/CheapestPriceUpdater.php

```diff
@@ -75,13 +81,23 @@ public function update(array $parentIds, Context $context): void
                     continue;
                 }
 
+                if ($variantId !== $productId) {
+                    $variantIdsUpdated[] = $variantId;
+                }
+
                 $accessorQuery->execute([
                     'accessor' => $accessor,
                     'id' => Uuid::fromHexToBytes($variantId),
                     'version' => $versionId,
                 ]);
             }
         }
+
+        if (!empty($variantIdsUpdated)) {
+            $this->dispatcher->dispatch(new ProductIndexerEvent($variantIdsUpdated, $context, [
+                SeoUrlUpdateListener::PRODUCT_SEO_URL_UPDATER,
```

Comment:
couldn't we skip more? :thinking:

Reply:
certainly, but currently I only see there is one skip event in `\Shopware\Storefront\Framework\Seo\SeoUrlRoute\SeoUrlUpdateListener::updateProductUrls`  so I don't know if we have any other events